inlong-sort/sort-connectors/elasticsearch-7/pom.xml

```diff
@@ -248,6 +248,10 @@
                                     <pattern>com.github.mustachejava</pattern>
                                     <shadedPattern>org.apache.flink.elasticsearch7.shaded.com.github.mustachejava</shadedPattern>
                                 </relocation>
+                                <relocation>
+                                    <pattern>org.apache.inlong.sort.elasticsearch</pattern>
+                                    <shadedPattern>org.apache.inlong.sort.elasticsearch7</shadedPattern>
```

Comment:
Dose the es6 also need to change pom?

Reply:
es6 conflict with es7, since es7 has relocated , es6 doesn't need  to do so